test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir

```diff
@@ -326,7 +326,7 @@ circuit Top :
     ; CHECK-NEXT:       logic [3:0] uint;
     ; CHECK:          endinterface
 
-    ; YAML:           FILE "Wire{{[/\]}}firrtl/gct/gct.yaml"
+    ; YAML:           FILE "Wire{{[/\]}}firrtl{{[/\]}}gct{{[/\]}}gct.yaml"
```

Comment:
Mega-nit: There's multiple styles in use here: (1) `{{.}}`, (2) `{{[/\]}}`, and (3) `{{/|\\\\}}`. Do we care about consistency across tests?

Reply:
I moved all (1) to (2), and (3) is different than the first two since it represents `/` on Linux and `\\` on Windows (there is 2 levels of escaping happening when printing StringAttrs). Also in case anyone following along is wondering, the horrible looking `{{/|\\\\}}` is used pretty commonly in LLVM.